client/src/containers/widget/empty-widget/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from "react";
+
+import Title from "@/components/ui/title";
+
+const EmptyWidget: FC<{ indicator: string; question?: string }> = ({
+  indicator,
+  question,
+}) => {
+  return (
+    <>
+      <header className="space-y-2 p-6">
+        <Title as="h3" className="text-base">
+          {indicator}
+        </Title>
+        {question && (
+          <p className="text-xs text-muted-foreground">{question}</p>
+        )}
+      </header>
+      <div className="flex h-full flex-col items-center justify-center gap-2">
+        <p className="font-semibold">No data</p>
+        <p className="text-xs">
+          There was no data found for this visualization
+        </p>
+      </div>
```

Comment:
To me, this is actually the `NoData` component: just a text informing about it. You don't need to replicate a whole widget to display a no data notice, you can just do `isEmptyData(data) ? <NoData /> : <HorizontalBarChart />` inside the "body" of the widget, and you don't need to replicate the rest of the skeleton.